### PR TITLE
4.3.0-release-notes

### PR DESCRIPTION
**Pull request by hmottestad**
Javadocs, release-notes and news item for 4.3.0

### CODE REVIEW

File: site/content/news/rdf4j-430.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+categories: ["news"]
+---
+RDF4J 4.3.0 is now available. This is a minor release fixing 34 issues.
+
+For more details, have a look at the [release notes](/release-notes/4.3.0).
+<!--more-->
+### Links
+
+- [Download RDF4J](/download/)
+- [release notes](/release-notes/4.3.0).
```
Comment (hmottestad) [inline site/content/news/rdf4j-430.md:14]: News item

File: site/content/release-notes/4.3.0.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ - Improvements to the extended query evaluation [1](https://github.com/eclipse/rdf4j/issues/635), [2](https://github.com/eclipse/rdf4j/issues/3947) 
+ - Several performance improvements [1](https://github.com/eclipse/rdf4j/issues/3798), [2](https://github.com/eclipse/rdf4j/pull/3799), [3](https://github.com/eclipse/rdf4j/issues/4435), [4](https://github.com/eclipse/rdf4j/issues/4212) 
+
+For a complete overview, see [all issues fixed in 4.3.0](https://github.com/eclipse/rdf4j/milestone/89?closed=1).
+
+There are no more planned minor releases for 4.x.x. The next planned release is 5.0.0.
+
+## Acknowledgements
+
+This release was made possible by contributions from Andreas Schwarte, Håvard M. Ottestad, Erik Godding Boye, Jeen Broekstra, Jerven Bolleman, Bart Hanssens, Wolfgang Schell, Mike Glazer, Patrick Wyler and Antoine Willerval.
```
Comment (hmottestad) [inline site/content/release-notes/4.3.0.md:27]: Release notes

File: site/content/release-notes/4.3.0.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+**A few notable features**
+- Solr, Lucene and Elasticsearch
+    - Solr and Lucene are upgraded to 8.9.0 and Elasticsearch is upgraded to 7.15.2
+    - Support for [complex Lucene queries](https://github.com/eclipse/rdf4j/pull/4196)
+    - Support for [customizing the query analyzer in Lucene and Elasticsearch](https://github.com/eclipse/rdf4j/issues/4235)
+ - SHACL
+   - A standalone SHACL Validator
+   - User defined [SPARQL queries](https://github.com/eclipse/rdf4j/issues/4489) in SHACL Constraints and Targets
+   - Support for sequence paths and alternate paths
+ - Migration of all our tests to Junit 5, thanks to Erik Godding Boye
```
Comment (erikgb) [inline site/content/release-notes/4.3.0.md:16]:
```suggestion
 - Migration of more tests to Junit 5
```
As there are still Junit 4 references in our codebase.

File: site/content/release-notes/4.3.0.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ - Remove dependency to javax.xml.bind from core libraries
```
Comment (erikgb) [inline site/content/release-notes/4.3.0.md:17]:
```suggestion
 - Remove Java EE dependency from core libraries
```

**Pull request merged**